src/Module.Server/Common/Models/CrpgAgentApplyDamageModel.cs

```diff
@@ -195,6 +195,34 @@ public override float GetDamageMultiplierForBodyPart(BoneBodyPartType bodyPart,
         defenderStunMultiplier = 1f;
     }
 
+    public override bool DecideCrushedThrough(Agent attackerAgent, Agent defenderAgent, float totalAttackEnergy, Agent.UsageDirection attackDirection, StrikeType strikeType, WeaponComponentData defendItem, bool isPassiveUsage)
+    {
+        EquipmentIndex wieldedItemIndex = attackerAgent.GetWieldedItemIndex(Agent.HandIndex.OffHand);
+        if (wieldedItemIndex == EquipmentIndex.None)
+        {
+            wieldedItemIndex = attackerAgent.GetWieldedItemIndex(Agent.HandIndex.MainHand);
+        }
+
+        WeaponComponentData? weaponComponentData = (wieldedItemIndex != EquipmentIndex.None) ? attackerAgent.Equipment[wieldedItemIndex].CurrentUsageItem : null;
```

Comment:
Can you break that into several lines
```suggestion
        var weaponComponentData = wieldedItemIndex != EquipmentIndex.None
            ? attackerAgent.Equipment[wieldedItemIndex].CurrentUsageItem
            : null;
```